lib/index.js

```diff
@@ -11,6 +11,7 @@ var crypto = require('crypto');
 var database = require('bedrock-mongodb');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
+require('bedrock-server');
```

Comment:
@dlongley, This was not added by me; however, it's there for creating the publicKeyId.  Currently, the publicKeyId is created using baseUri, so that's why bedrock-server was added.  Can we get around that?  
https://github.com/digitalbazaar/bedrock-key/blob/master/lib/index.js#L82